src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
Great job @kamranahmad123 🤝

- [ ] I would recommend kindly deleting this `setupTests.js` file from your project as it is not being used. This will make your project conform to the ducks' pattern.

Reply:
- [ ] Well resolved 🥇